torch/linalg/__init__.py

```diff
@@ -286,7 +286,10 @@
 Args:
     input (Tensor): The input tensor. If dim is None, x must be 1-D or 2-D, unless :attr:`ord`
         is None. If both :attr:`dim` and :attr:`ord` are None, the 2-norm of the input flattened to 1-D
-        will be returned.
+        will be returned. Its data type must be either a floating point or complex type. For complex
+        inputs, the norm is calculated on of the absolute values of each element. If the input is
+        complex and neither :attr:`dtype` nor :attr:`out` is specified, the result's data type will be
+        the corresponding downgraded real number type.
```

Comment:
This sentence needs to be updated to be consistent with the torch/functional.py documentation.

Reply:
Does the torch.linalg.cond documentation need a similar update, too?

Reply:
Updated this and `torch.linalg.cond`